nakadi/kafka_pool.py

```diff
@@ -59,6 +60,8 @@ def kafka_client(self):
                 logging.warning("Exception when closing kafka client: %s", e)
             raise kafka_error
         except:
+            logging.warning("Exception occured while kafka client was out of pool:")
+            traceback.print_exc()
```

Comment:
oh, what are you trying to do here? Your could catch the exception using the following syntax:

```python
try:
  ...
except Exception as e:
  ...
```